holmes/plugins/toolsets/json_filter_mixin.py

```
@@ -0,0 +1,122 @@
+        ),
+        "jsonpath": ToolParameter(
+            description="Optional jsonpath expression to extract specific parts of the JSON.",
+            type="string",
+            required=False,
+        ),
+        "jq": ToolParameter(
+            description="Optional jq filter to apply to the JSON.",
+            type="string",
+            required=False,
+        ),
+    }
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)